CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ The changes are relative to the previous release, unless the baseline is specifi
 * Add avif(Un)SignedFraction structs and avifDoubleTo(Un)SignedFraction
   utility functions.
 
+### Added since 1.1.1
```

Comment:
Nit: "Added" goes before "Changed". The order is Added/Changed/Removed. See https://github.com/AOMediaCodec/libavif/wiki/Release-Checklist